filter-effects-2/Overview.bs

```diff
@@ -78,6 +78,10 @@ output as its SourceGraphic.
 
 Filter functions must operate in the sRGB color space.
 
+If the filter functions list includes a <a href="https://drafts.fxtf.org/filter-effects-1/#blurEquivalent">blur()</a>
```

Comment:
Is it just blur() that is pixel-moving?

Reply:
For shorthand filters, both `blur()` and `drop-shadow()` move pixels.  The filter list can also include references to SVG filters, which include various ways of extending effects to neighbouring pixels.

But I don't think it's worth making special cases for the other situations.

Reply:
I added a parenthetical "(or any pixel moving filter)" - let me know if that works for you.